### ISSUE TITLE
rasa.nlu.model.UnsupportedModelError: The model version is to old to be loaded by this Rasa NLU instance. Either retrain the model, or run withan older version. Model version: 1.2.3 Instance version: 1.3.0a1

### ISSUE THREAD

**Issue by ngochuyenluu** [RasaHQ/rasa#4362]
**Rasa version**:1.2.5

**Rasa X version** (if used & relevant):0.20.1

**Python version**:3.6.8

**Operating system** (windows, osx, ...): Linux

**Issue**: rasa.nlu.model.UnsupportedModelError: The model version is to old to be loaded by this Rasa NLU instance. Either retrain the model, or run withan older version. Model version: 1.2.3 Instance version: 1.3.0a1

**Error (including full traceback)**:
```
Starting chabot_hotline_rasa_1 ... done
Attaching to chabot_hotline_rasa_1
rasa_1  | 2019-08-30 14:47:44 INFO     root  - Starting Rasa server on http://localhost:5005
rasa_1  | 2019-08-30 14:47:45 ERROR    rasa.core.agent  - Could not load model due to The model version is to old to be loaded by this Rasa NLU instance. Either retrain the model, or run withan older version. Model version: 1.2.3 Instance version: 1.3.0a1.
rasa_1  | Experienced exception while trying to serve
rasa_1  | Traceback (most recent call last):
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/sanic/app.py", line 1096, in run
rasa_1  |     serve(**server_settings)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/sanic/server.py", line 742, in serve
rasa_1  |     trigger_events(before_start, loop)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/sanic/server.py", line 604, in trigger_events
rasa_1  |     loop.run_until_complete(result)
rasa_1  |   File "uvloop/loop.pyx", line 1417, in uvloop.loop.Loop.run_until_complete
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/run.py", line 233, in load_agent_on_start
rasa_1  |     action_endpoint=endpoints.action,
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/agent.py", line 258, in load_agent
rasa_1  |     remote_storage=remote_storage,
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/agent.py", line 921, in load_local_model
rasa_1  |     remote_storage=remote_storage,
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/agent.py", line 368, in load
rasa_1  |     interpreter = NaturalLanguageInterpreter.create(nlu_model)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/interpreter.py", line 57, in create
rasa_1  |     return RasaNLUInterpreter(model_directory=obj)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/interpreter.py", line 263, in __init__
rasa_1  |     self._load_interpreter()
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/interpreter.py", line 286, in _load_interpreter
rasa_1  |     self.interpreter = Interpreter.load(self.model_directory)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/nlu/model.py", line 300, in load
rasa_1  |     Interpreter.ensure_model_compatibility(model_metadata)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/nlu/model.py", line 275, in ensure_model_compatibility
rasa_1  |     "".format(model_version, rasa.__version__)
rasa_1  | rasa.nlu.model.UnsupportedModelError: The model version is to old to be loaded by this Rasa NLU instance. Either retrain the model, or run withan older version. Model version: 1.2.3 Instance version: 1.3.0a1
rasa_1  | Traceback (most recent call last):
rasa_1  |   File "/usr/local/bin/rasa", line 10, in <module>
rasa_1  |     sys.exit(main())
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/__main__.py", line 76, in main
rasa_1  |     cmdline_arguments.func(cmdline_arguments)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/cli/run.py", line 119, in run
rasa_1  |     rasa.run(**vars(args))
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/run.py", line 55, in run
rasa_1  |     **kwargs
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/run.py", line 194, in serve_application
rasa_1  |     app.run(host="0.0.0.0", port=port, ssl=ssl_context)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/sanic/app.py", line 1096, in run
rasa_1  |     serve(**server_settings)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/sanic/server.py", line 742, in serve
rasa_1  |     trigger_events(before_start, loop)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/sanic/server.py", line 604, in trigger_events
rasa_1  |     loop.run_until_complete(result)
rasa_1  |   File "uvloop/loop.pyx", line 1417, in uvloop.loop.Loop.run_until_complete
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/run.py", line 233, in load_agent_on_start
rasa_1  |     action_endpoint=endpoints.action,
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/agent.py", line 258, in load_agent
rasa_1  |     remote_storage=remote_storage,
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/agent.py", line 921, in load_local_model
rasa_1  |     remote_storage=remote_storage,
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/agent.py", line 368, in load
rasa_1  |     interpreter = NaturalLanguageInterpreter.create(nlu_model)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/interpreter.py", line 57, in create
rasa_1  |     return RasaNLUInterpreter(model_directory=obj)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/interpreter.py", line 263, in __init__
rasa_1  |     self._load_interpreter()
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/core/interpreter.py", line 286, in _load_interpreter
rasa_1  |     self.interpreter = Interpreter.load(self.model_directory)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/nlu/model.py", line 300, in load
rasa_1  |     Interpreter.ensure_model_compatibility(model_metadata)
rasa_1  |   File "/usr/local/lib/python3.6/site-packages/rasa/nlu/model.py", line 275, in ensure_model_compatibility
rasa_1  |     "".format(model_version, rasa.__version__)
rasa_1  | rasa.nlu.model.UnsupportedModelError: The model version is to old to be loaded by this Rasa NLU instance. Either retrain the model, or run withan older version. Model version: 1.2.3 Instance version: 1.3.0a1

```

**Command or request that led to error**:
```
docker-compose up
```
Hello, I installed rasa ver 1.2.5 and it works perfectly on my local sever. But when I tried to run docker, I have some errors that I committed issue. As the recommendation in rasa tutorial, I tried to run docker-compose and it raises this error. I have read a lot of posts in rasa community about this problem. Some solutions are uninstall rasa and reinstall rasa==0.15.0a6 (I tried, it didn't work) or downgrade rasa to 1.0 and train rasa_nlu. As I understand, rasa is combined between rasa_nlu and rasa_core, I don't want to train them separately. Is there anyone have same problem with me when run docker-compose?

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @JustinaPetr will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by erohmensing** [comment]
@ngochuyenluu what `rasa` image are you using in the docker-compose? Is it set to `latest`? Instance version `1.3.0a1` isn't released. If so, setting it to `stable` should do the trick.

**Comment by ngochuyenluu** [comment]
Hello @erohmensing, thank for responding. I added in the ````docker-compose.yml``` like this:
````
version: '3.0'
services:
  rasa:
    image: rasa/rasa:latest-full
    ports:
      - 5005:5005
    volumes:
      - ./:/app
    command:
      - run
````
So I suppose the rasa image is set to latest. If I understand well your recommendation, we will change rasa/rasa:latest-full to rasa/rasa:stable? I have tried, but it doesn't work. Did I misunderstanding or anything? Thank you in advance.

**Comment by erohmensing** [comment]
Sorry, I misspoke, there isn't a stable tag for rasa it seems. Just set it to the most recent released version, in this case 1.2.7 -- so `rasa/rasa:1.2.7-full`

**Comment by ngochuyenluu** [comment]
Hello, it worked. Thank you a lot :)
